microprofile.jdt/com.redhat.microprofile.jdt.test/src/main/java/com/redhat/microprofile/jdt/core/ProjectLabelTest.java

```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
+* Copyright (c) 2020 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.microprofile.jdt.core;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.jdt.core.IJavaProject;
+import org.junit.After;
+import org.junit.Test;
+
+import com.redhat.microprofile.commons.ProjectLabelInfoEntry;
+import com.redhat.microprofile.jdt.core.BasePropertiesManagerTest.GradleProjectName;
+import com.redhat.microprofile.jdt.core.BasePropertiesManagerTest.MavenProjectName;
+import com.redhat.microprofile.jdt.core.utils.JDTMicroProfileUtils;
+import com.redhat.microprofile.jdt.internal.core.JavaUtils;
+
+import org.junit.Assert;
+
+/**
+ * Project label tests
+ *
+ */
+public class ProjectLabelTest {
+	
+	@After
+	public void destroy() throws CoreException {
+		// clear all projects from workspace
+		ResourcesPlugin.getWorkspace().getRoot().delete(true, null);
```

Comment:
Why did you that? Could you try to comment this code and see if travis is working better please.